src/libraries/System.Text.RegularExpressions/tests/RegexParserTests.cs

```diff
@@ -895,21 +882,18 @@ private static void Throws(RegexParseError error, Action action)
             {
                 action();
             }
-            catch (ArgumentException e)
+            catch (RegexParseException e)
```

Comment:
Now that this method doesn't run on Framework, can you also delete the `If (PlatformDetection.IsNetFramework)` block above this?

Reply:
Agreed. Done.